src/core/crypto_print.c

```diff
@@ -67,34 +67,6 @@ void Crypto_tmPrint(TM_t *tm_frame)
     printf("Current TM in memory is: \n");
```

Comment:
What is going on in this print statement.  Should we just remove it entirely at this point?